openmetadata-ui/src/main/resources/ui/src/rest/DocStoreAPI.ts

```diff
@@ -31,7 +32,9 @@ export const getAllKnowledgePanels = async (params: { fqnPrefix: string }) => {
 };
 
 export const getDocumentByFQN = async (fqn: string) => {
-  const response = await axiosClient.get<Document>(`${BASE_URL}/name/${fqn}`);
+  const response = await axiosClient.get<Document>(
+    `${BASE_URL}/name/${getEncodedFqn(fqn)}`
```

Comment:
As we are adding encoding here, make sure to check places where getDocumentByFQN is used and remove the encoding from there.

Reply:
Thanks for reminding, missed in one place. will update there